src/components/PrintDimensionCards.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import "../styles/PrintCards.css";
+import "../styles/Footer.css";
+import { Dimension as DimensionType } from "../types/dimension";
+
+import { Slider } from "antd";
+
+import "../styles/PrintDimension.css";
+
+const PRACTICE_BG_COLOUR = "#ffc4d3";
+const BELIEF_BG_COLOUR = "#c4ddff";
+
+interface PrintFullDimensionCardsProps {
+  passAllDimensions: DimensionType[];
+}
+
+interface PrintDimensionProps {
+  dimensionValue: number;
+  secondStatement: boolean;
+  allDimensions: DimensionType;
+}
+
+class PrintDimensionCards extends React.Component<
```

Comment:
would it be possible to change this to a functional component rather than a class? (consistency)

Reply:
@PreetPatel I think @TCHE614 did this PR for print cards. I assume it may have been issues with refs which led to him using classes instead of functional components?